.build/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Initialize flags for config.json and private key
+CONFIG_EXISTS=false
+PK_EXISTS=false
+
+# Check if config.json exists
+if [ -f /app/config.json ]; then
+  echo "config.json detected"
+  CONFIG_EXISTS=true
+else
+  echo "config.json not found"
+fi
+
+# Check if data-proxy-private-key.json exists
+if [ -f /app/data-proxy-private-key.json ]; then
+  echo "data-proxy-private-key.json detected"
+  PK_EXISTS=true
+elif [ -n "$SEDA_DATA_PROXY_PRIVATE_KEY" ]; then
+  # If private key file does not exist, check if the private key is provided via environment variable
+  echo "Private key provided via environment variable"
+  echo "$SEDA_DATA_PROXY_PRIVATE_KEY" >/app/data-proxy-private-key.json
+  PK_EXISTS=true
+else
+  echo "No private key provided"
+fi
+
+run_bun_command() {
+  if ! bun "$@"; then
+    echo "Failed to run: bun $*"
+    exit 1
+  fi
+}
+
+# Determine the command to run based on the presence of config.json and private key
+if [ "$CONFIG_EXISTS" = true ] && [ "$PK_EXISTS" = true ]; then
+  # Both config.json and private key are provided
+  echo "Running with config and private key"
+  RUN_CMD="start run --config /app/config.json --private-key-file /app/data-proxy-private-key.json"
+elif [ "$CONFIG_EXISTS" = true ] && [ "$PK_EXISTS" = false ]; then
+  # Only config.json is provided
+  echo "Running with config only"
+  run_bun_command start init
+  RUN_CMD="start run --config /app/config.json"
+else
+  # Neither config.json nor private key is provided
+  echo "Running with --disable-proof"
+  run_bun_command start init
+  RUN_CMD="start run --disable-proof"
```

Comment:
This doesn't make sense: `--disable-proof` should only skip validating whether the incoming request has the correct proof, it doesn't deal with the private key or config at all.

Reply:
The data-proxy can not run when there is no config or no private key (file). `--disable-proof` only disables the request verification, so it should be a completely separate option.

I think the scenarios should be covered by the binary itself where it will error if it is unable to find a config and/or private key (file).

If someone wants to run it in debug mode they should pass the `--disable-proof` flag manually.

Reply:
yes, passing it manually makes sense, maybe we could have an container env variable like `FLAGS` or so..